### PR TITLE
[TACHYON-1428] Documented parts of the tachyon.shell.command package in preparation for the checkstyle rule

### PR DESCRIPTION
**Pull request by HorizonNet**
https://tachyon.atlassian.net/browse/TACHYON-1428

### CODE REVIEW

File: shell/src/main/java/tachyon/shell/command/CreateLineageCommand.java [hunk 1]
```diff
@@ -31,10 +31,16 @@
 import tachyon.job.JobConf;
 
 /**
- * TODO(yupeng): add javadoc
+ * Creates a new lineage for given input and output file names with optional exclusion parameters.
```
Comment (aaudiber) [inline shell/src/main/java/tachyon/shell/command/CreateLineageCommand.java]: Creates a lineage for the given input files, output files, and command line job.

File: shell/src/main/java/tachyon/shell/command/FileInfoCommand.java [hunk 2]
```diff
@@ -30,6 +30,12 @@
  */
 public final class FileInfoCommand extends WithWildCardPathCommand {
 
+  /**
+   * Constructs a new instance to display the file's all block info.
```
Comment (aaudiber) [inline shell/src/main/java/tachyon/shell/command/FileInfoCommand.java]: to display the block info for the file's blocks

**Comment by aaudiber** [comment]
@HorizonNet Thanks for the PR, I left a couple comments

**Comment by HorizonNet** [comment]
@aaudiber Thanks for the comments. I just updated the PR.

**Comment by aaudiber** [comment]
+1 LGTM

**Pull request merged**